openupgrade_scripts/scripts/account/14.0.1.1/post-migration.py

```diff
@@ -0,0 +1,383 @@
+# Copyright 2021 ForgeFlow S.L.  <https://www.forgeflow.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+from odoo.tools.translate import _
+
+
+def fill_account_journal_posted_before(env):
+    openupgrade.logged_query(
+        env.cr,
+        """
+        UPDATE account_move
+        SET posted_before = TRUE
+        WHERE state = 'posted'""",
+    )
+
+
+def fill_code_prefix_end_field(env):
+    openupgrade.logged_query(
+        env.cr,
+        """
+        UPDATE account_group
+        SET code_prefix_end = code_prefix_start
+        """,
+    )
+
+
+def fill_default_account_id_field(env):
+    openupgrade.logged_query(
+        env.cr,
+        """
+        UPDATE account_journal
+        SET default_account_id = {0}
+        WHERE {0} = {1} OR ({0} IS NOT NULL AND {1} IS NULL);
+        UPDATE account_journal
+        SET default_account_id = {1}
+        WHERE {0} IS NULL AND {1} IS NOT NULL
+        """.format(
+            openupgrade.get_legacy_name("default_credit_account_id"),
+            openupgrade.get_legacy_name("default_debit_account_id"),
+        ),
+    )
+
+
+def fill_payment_id_and_statement_line_id_fields(env):
+    openupgrade.logged_query(
+        env.cr,
+        """
+        UPDATE account_move_line aml
+        SET payment_id = am.payment_id
+        FROM account_move am
+        WHERE am.id = aml.move_id AND am.payment_id IS NOT NULL
+        """,
+    )
+    openupgrade.logged_query(
+        env.cr,
+        """
+        UPDATE account_move_line aml
+        SET statement_line_id = am.statement_line_id
+        FROM account_move am
+        WHERE am.id = aml.move_id AND am.statement_line_id IS NOT NULL
+        """,
+    )
+
+
+def fill_active_field_account_reconcile_model(env):
+    openupgrade.logged_query(
+        env.cr,
+        """
+        UPDATE account_reconcile_model
+        SET active = TRUE
```

Comment:
You don't have to set the value here manually, because the field has this exact same value as its default, and if a field has a default the ORM will set it on new fields on existing models. This is why the analysis mentions `hasdefault` as a hint that perhaps the column does not need to be populated manually.

Reply:
done